src/main/resources/Dockerfile

```diff
@@ -23,6 +23,8 @@ ARG TARANTOOL_DATADIR="/tmp/data"
 ARG TARANTOOL_LOGDIR="/tmp/log"
 ARG TARANTOOL_INSTANCES_FILE="./instances.yml"
 ARG TARANTOOL_CLUSTER_COOKIE="testapp-cluster-cookie"
+ARG START_DELAY="5s"
```

Comment:
It probably doesn't work 
<img width="216" alt="image" src="https://github.com/tarantool/testcontainers-java-tarantool/assets/22258443/c8cce979-7906-4dbb-8e3a-81897d2aa479">

Reply:
Will recheck it

Reply:
It works on linux environment

Reply:
This works for building Dockerfile from Java and ```docker build``` command